climada/engine/impact.py

```diff
@@ -939,6 +946,152 @@ def _cen_return_imp(imp, freq, imp_th, return_periods):
         imp_fit[wrong_inten] = 0.
 
         return imp_fit
+    
+
+    def select(self,
+               event_ids=None, event_names=None, dates=None,
+               coord_exp=None):
+        """
+        Select a subset of events and/or exposure points from the impact.
+        If multiple input variables are not None, it returns all the impacts
+        matching at least one of the conditions.
```

Comment:
I was thinking of this too. For the first three attributes `event_id`, `event_name` and `dates` it makes little sense to me, as they are all unique event identifiers, hence I think OR is more natural than AND. For mixing `event_id` `event_name` or `dates`, the AND makes more sense. 

Not sure what to do. 

One solution would be to make an additional method `append` which would allow to make sequential `select` - `append` to obtain the OR version.